### PR TITLE
[UI] Redesign chat list page

### PR DESCRIPTION
**Pull request by Solpio**

### CODE REVIEW

File: admin-app/src/pages/main-page/index.tsx [hunk 1]
```diff
@@ -114,8 +113,18 @@ export const MainPage = () => {
 
   return (
     <>
+      <div
```
Comment (kurkul608) [inline admin-app/src/pages/main-page/index.tsx]: use style components

File: admin-app/src/pages/main-page/index.tsx [hunk 2]
```diff
@@ -114,8 +113,18 @@ export const MainPage = () => {
...
+          gridColumnEnd: -1,
+          fontSize: "2rem",
+          color: "#ffffff",
+        }}
+      >
+        Chats
+      </div>
       <ChatListWidget />
-      <SendMessageWidget />
+      {/*<SendMessageWidget />*/}
```
Comment (kurkul608) [inline admin-app/src/pages/main-page/index.tsx]: remove

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx [hunk 3]
```diff
@@ -12,36 +10,66 @@ import {
...
 import { IChatInterface } from "../../../../interfaces/chat.interface";
+import { SettingsSvg } from "../settings";
 
 export const ChatListWidget = () => {
   const { list } = useAppSelector((state) => state.chats);
   const dispatch = useAppDispatch();
   useEffect(() => {
     dispatch(getAllChats());
   }, []);
+  console.log(list);
```
Comment (kurkul608) [inline admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: remove

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx [hunk 4]
```diff
@@ -12,36 +10,66 @@ import {
...
   const navigate = useNavigate();
   const handleOnClick = useCallback(
     (chat: IChatInterface) => navigate(`/chat/${chat.id}`, { replace: true }),
     [navigate]
   );
+  console.log(list[0]);
```
Comment (kurkul608) [inline admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: remove

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx [hunk 5]
```diff
@@ -12,36 +10,66 @@ import {
...
-      <ChatListWrapper>
-        <Widget name={"Chat list widget"}>
-          <ChatListUL>
-            {list.map((chat) => (
-              <li key={`widget-chat-list--${chat.id}`}>
-                <Chat onClick={() => handleOnClick(chat)}>
-                  <ChatPhotoWrapper>
-                    <ChatPhoto>{chat.title[0].toUpperCase()}</ChatPhoto>
-                  </ChatPhotoWrapper>
+      {/*<ChatListWrapper>*/}
```
Comment (kurkul608) [inline admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: remove comments

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx [hunk 6]
```diff
@@ -12,36 +10,66 @@ import {
...
+                style={{
+                  display: "flex",
+                  alignItems: "center",
+                  gap: 20,
+                }}
+              >
+                <ChatPhotoWrapper>
+                  <ChatPhoto>{chat.title[0].toUpperCase()}</ChatPhoto>
+                </ChatPhotoWrapper>
+                <div>
```
Comment (kurkul608) [inline admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: better use style components

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx [hunk 7]
```diff
@@ -12,36 +10,66 @@ import {
...
                   <ChatTitle>{chat.title}</ChatTitle>
-                </Chat>
-              </li>
-            ))}
-          </ChatListUL>
-        </Widget>
-      </ChatListWrapper>
+                  <span>32k subscribers</span>
```
Comment (kurkul608) [inline admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: use SC

File: admin-app/src/shared/chat-list/components/settings/index.tsx [hunk 8]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export const SettingsSvg = () => {
+  return (
+    <svg
```
Comment (kurkul608) [inline admin-app/src/shared/chat-list/components/settings/index.tsx:5]: svg must be in files with the extension .svg

File: admin-app/src/shared/layout/index.tsx [hunk 9]
```diff
@@ -1,17 +1,19 @@
...
 import { Aside } from "./aside";
-import { Outlet } from "react-router-dom";
+import { Outlet, useLocation } from "react-router-dom";
 import React from "react";
 import { Main } from "./styled";
 import { widgetSize } from "../../constants/size";
 
 export const Layout = () => {
+  const location = useLocation();
+  const autoFlow = location.pathname !== "/";
```
Comment (kurkul608) [inline admin-app/src/shared/layout/index.tsx]: Look at your personal messages

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx (hunk 6 above)
Comment (Solpio) [reply admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: Why is it better to use style components here if we don't need any styles?

File: admin-app/src/shared/chat-list/components/settings/index.tsx (hunk 8 above)
Comment (Solpio) [reply admin-app/src/shared/chat-list/components/settings/index.tsx:5]: I wanted pass props here to change color of the setting svg. Is it works when we import svg as file.svg?

Comment (kurkul608) [reply admin-app/src/shared/chat-list/components/settings/index.tsx:5]: use css, like .svg {path { stroke: #FFFFFF }}

File: admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx (hunk 6 above)
Comment (kurkul608) [reply admin-app/src/shared/chat-list/components/chat-list-widget/index.tsx]: Okay, I agree with you.

**Pull request merged**